maintenance/create_lut.py

```diff
@@ -0,0 +1,45 @@
+import json
```

Comment:
I don't fully understand how this script is run - is this for local execution? Because as far as I see it is not called from anything else atm.

Reply:
only ever run locally. I added this only for reference. This will go one we switch to making use of th eindexing service